### DISCUSSION TITLE
Chrome OS Flex

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [sebanc/brunch#1444] [Q&A] [answered]
Even after the stable version of [Chrome OS Flex](https://chromeunboxed.com/chrome-os-flex-free-windows-pc-macbook-mac-os/) is released and support for the Google Play Store is added, will the development of Brunch be continued?

Thanks in advance!

**Comment by MaxHeadrO0m** [comment]
FYI [Chrome OS Flex installation guide](https://support.google.com/chromeosflex/answer/11541904?hl=en&ref_topic=11551271)

**Comment by kristibektashi** [comment, marked as answer]
[Chrome OS Flex will not support the Play Store or Android Apps](https://support.google.com/chromeosflex/answer/11547280?hl=en&ref_topic=11618314)

**Reply by JuliaAgostino** [reply]
There's nothing to suggest that Android applications won't be supported in the near future.

**Reply by benjamin920101** [reply]
Maybe you can try to replace ROOT-A partition file

**Reply by JuliaAgostino** [reply]
> > There's nothing to suggest that Android applications won't be supported in the near future.
> 
> 
> 
> Google won't support the Play Store on ChromeOS Flex because every device that has the Play Store needs to be manually certified by Google, Most phones (excluding certain brands) and Chromebooks are certified by Google to use the Play Store. However, ChromeOS Flex cannot be certified because it can be installed on any device, (same reason Cloudready didn't support the Play Store)

Unless you work at Google, that's just hearsay.
